hugegraph-core/src/main/java/org/apache/hugegraph/traversal/algorithm/EdgeExistenceTraverser.java

```diff
@@ -0,0 +1,48 @@
+package org.apache.hugegraph.traversal.algorithm;
+
+import org.apache.hugegraph.HugeGraph;
+import org.apache.hugegraph.backend.id.Id;
+import org.apache.hugegraph.backend.query.ConditionQuery;
+import org.apache.hugegraph.type.HugeType;
+import org.apache.hugegraph.type.define.Directions;
+import org.apache.hugegraph.type.define.HugeKeys;
+import org.apache.tinkerpop.gremlin.structure.Edge;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class EdgeExistenceTraverser extends HugeTraverser {
+    public EdgeExistenceTraverser(HugeGraph graph) {
+        super(graph);
+    }
+
+    public Iterator<Edge> queryEdgeExistence(
+        Id sourceId, Id targetId,
+        String label, long limit) {
+        if ("BOTH".equals(label)) {
+            return queryByNeighbor(sourceId, targetId, limit);
+        }
+        ConditionQuery conditionQuery = new ConditionQuery(HugeType.EDGE);
+        conditionQuery.eq(HugeKeys.OWNER_VERTEX, sourceId);
+        conditionQuery.eq(HugeKeys.OTHER_VERTEX, targetId);
+        conditionQuery.eq(HugeKeys.LABEL, getEdgeLabelId(label));
+        conditionQuery.eq(HugeKeys.DIRECTION, Directions.OUT);
+        conditionQuery.eq(HugeKeys.SORT_VALUES, "");
```

Comment:
please check if exists sortkey in the edge label, fallback if yes

Reply:
> please check if exists sortkey in the edge label, fallback if yes

How exactly is the fallback implemented? I'm not quite sure which aspects of ConditionQuery can utilize EdgeLabel.sortkeys.

Reply:
> please check if exists sortkey in the edge label, fallback if yes

Hi @javeme ， how exactly is the fallback implemented? I'm not quite sure which aspects of ConditionQuery can utilize EdgeLabel.sortkeys.